### PR TITLE
appveyor CI and x64

### PR DESCRIPTION
**Pull request by chcg**
- initial appveyor.yml CI config (prepared for github deployment on tagging)
- updated to VS2013
- added x64 configuration
- updated n++ files with data from https://github.com/npp-plugins/plugintemplate v3.1
- necessary minimal x64 adaptations
- updated sqlite to version 3.16.2 from https://sqlite.org/2017/sqlite-amalgamation-3160200.zip
- reduced compiler warnings

, see https://ci.appveyor.com/project/chcg/fingertext/build/0.5.61.5

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        }
+
+artifacts:
+  - path: FingerText_*.zip
+    name: releases
+
+deploy:
+    provider: GitHub
+    auth_token:
+        secure: !!TODO, see https://www.appveyor.com/docs/deployment/github/#provider-settings!!
```
Comment (chcg) [inline appveyor.yml]: token needed to enable github deployment

Comment (sigzero) [reply appveyor.yml:62]: It looks like you have a 64bit build almost completed except for this. Can you finish this out? This is a must-have plugin for Notepad++!

Comment (chcg) [reply appveyor.yml:62]: @sigzero This is something which just could be added by the project owner.

**Comment by Craigy-** [comment]
Great work! It's possible for you to upload compiled DLL? Please place a link here.

**Comment by chcg** [comment]
@Craigy- The compiled dlls were available in the past from appveyor but just for 3 month.
@erinata Any chance to get this merged? In the meantime n++ files would need to be updated and maybe also github actions might be a better choice instead of appveyor now.

**Pull request merged**